hoomd/hpmc/ShapeConvexPolygon.h

```diff
@@ -45,25 +45,25 @@ const unsigned int MAX_POLY2D_VERTS=64;
 */
 struct PolygonVertices : ShapeParams
     {
-    /// X coordinates of the vertices
```

Comment:
The `///` comments that precede a class/struct member variable or method should remain. These are special documentation comments that notify Doxygen (http://doxygen.nl/manual/docblocks.html) and other tools (such as IDEs) they are documentation.

I believe the comment requesting `///` being replaced with `//` was referring to a comment inside code that was not documenting a class or struct member.